lib/licenses.nix

```diff
@@ -442,6 +442,11 @@ lib.mapAttrs (n: v: v // { shortName = n; }) rec {
     fullName = "libpng License";
   };
 
+  libpng2 = {
+    fullName = "libpng License v2"; # 1.6.36+
+    inherit (libpng) url;
```

Comment:
Humm no that will point to the SPDX url of the previous/old license.
I think `url` should rather be https://sourceforge.net/p/libpng/code/ci/v1.6.36/tree/LICENSE or http://www.libpng.org/pub/png/src/libpng-LICENSE.txt

At least until SPDX adds a new entry or updates the `libpng` entry.

Reply:
Well the SPDX page itself links to http://www.libpng.org/pub/png/src/libpng-LICENSE.txt , but you're right the libpng spdx identifier as-is seems incorrect for now.  Bah.

I'll bump to point to the license directly? Coming shortly...